src/test/resources/org/bytedeco/javacpp/AdapterTest.h

```diff
@@ -10,9 +10,32 @@ std::wstring testStdWString(std::wstring str) {
     return str;
 }
 
-char *testCharString(const char *str) {
-    return strdup(str);
-    // memory leak...
```

Comment:
That's intentional to make sure that pointers with different values than the first argument also work correctly.